rpm/libtiff.spec

```diff
@@ -53,6 +53,7 @@ Man and info pages for %{name}.
 
 %build
 export CFLAGS="%{optflags} -fno-strict-aliasing"
+sed -i "s/for.*/for false/g" autogen.sh
```

Comment:
What is this line? this seems to be causing build failure later.

Reply:
Fixed.